### PR TITLE
doc: correct link reference in quick start guide

### PR DESCRIPTION
**Pull request by Sheetalpamecha**
Correct the link reference 
Signed-off-by: Sheetal Pamecha <spamecha@redhat.com>

### CODE REVIEW

File: docs/devel-quick-start.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ SPDX-License-Identifier: Apache-2.0
 1. Create a branch from where you want to base your work (usually main).
 1. Make your changes and arrange them in readable commits.
 1. Make sure your commit messages are in the proper format.
-1. Make sure all [tests](docs/testing.md) pass, and add any new [tests](docs/testing.md)
+1. Make sure all [tests](./testing.md) pass, and add any new [tests](./testing.md)
```
Comment (nirs) [inline docs/devel-quick-start.md:14]: The current link is not broken, try here:
https://github.com/RamenDR/ramen/blob/main/docs/devel-quick-start.md

Maybe you want to improve the links to use shorter form? Please improve the commit message to make your intent more clear.

Comment (raghavendra-talur) [reply docs/devel-quick-start.md:14]: Rest of the links are ok but this one is broken. My mistake when I refactored the devel-quick-start.md file.

Comment (Sheetalpamecha) [reply docs/devel-quick-start.md:14]: @nirs Thanks for reviewing. While accessing both the links I am getting this result.

Screenshot 2024-03-19 at 8 47 28 AM

**Review by raghavendra-talur — APPROVED** [review]

**Review by nirs — APPROVED** [review]
Thanks @Sheetalpamecha!

**Pull request merged**